src/metaswitch/ellis/prov_tools/utils.py

```diff
@@ -197,15 +197,18 @@ def delete_user(private_id, public_id, force=False):
 
     return success
 
-def display_user(public_id, short=False):
+# quiet flag will prevent any output to stdout. This way we can check that the
+# user exists and has valid xml, without swamping the output with large iFCs
+def display_user(public_id, short=False, quiet=False):
     success = True
     callback = Callback()
 
-    if not short:
+    if not short and not quiet:
         print "Public User ID %s:" % (public_id,)
```

Comment:
Can we have a custom print function that handles the `quiet` parameter?  Rather than wrapping every `print` in an `if`.